tests/dummy/app/templates/application.hbs

```diff
@@ -22,3 +22,10 @@ Datepicker already disabled before it's rendered:
 Inputless datepicker:
 {{pikaday-inputless onSelection=(action (mut someDate))}}
 {{someDate}}
+
+Inputless datepicker never go back:
```

Comment:
This doesn't seem like the most intuitive naming to me, what is it supposed to express?